orca-remote-stage/src/main/kotlin/com/netflix/spinnaker/orca/remote/stage/messages/messages.kt

```diff
@@ -0,0 +1,34 @@
+package com.netflix.spinnaker.orca.remote.stage.messages
+
+import com.netflix.spinnaker.orca.api.pipeline.models.ExecutionStatus
+
+interface RemoteStageMessage {
+  val stageExecutionId: String
+  val pipelineExecutionId: String
+}
+
+data class RemoteStageStatusMessage(
+  override val stageExecutionId: String,
+  override val pipelineExecutionId: String,
+  val status: ExecutionStatus,
+  val outputs: MutableMap<String, Any>,
+  val startTime: Long,
+  val endTime: Long,
```

Comment:
Let's represent time and durations as rich objects, rather than primitives... i.e. `Instant`.